platform/entanglement/src/actions/AbstractComposeAction.js

```diff
@@ -141,6 +141,8 @@ define(
                 currentParent
             ).then(function (newParentObj) {
                 return composeService.perform(object, newParentObj);
+            }, function () {
+                return Promise.reject({ message: "cancelled" });
```

Comment:
Good catch on this one, these cases need to be distinguished.

The `message` property of an Error is normally a human-readable string. (While this isn't an Error object, CopyAction presumes it will be, which is a useful convention for Promise rejection.) I think using this property to track error types could lead to mistakes under maintenance.

What about having a `cancelled` property on the action itself? A CopyAction is an AbstractComposeAction so it is reasonable to communicate information in this manner. `this.cancelled` seems easy to check and hard to misinterpret (note that you'll need to `bind` callbacks to `this` or similar.)